packages/core/core.ts

```diff
@@ -19,7 +19,7 @@ import * as hooking from "@jazzer.js/hooking";
 import { registerInstrumentor } from "@jazzer.js/instrumentor";
 
 export interface Options {
-	fuzzTarget: string;
+	fuzzEntryPoint: string;
```

Comment:
WDYT about renaming only this property and leaving the externally / user visible name as is? Fuzz target is a commonly used term and I wonder if we would confuse users by introducing another one.

Reply:
I'm torn on this. I wanted to keep documentation and code consistent with each other. That said, I find the naming between `fuzzTarget` and `fuzzFunction` in the `options` interface misleading in the first place. IMO, `fuzzTarget` and `fuzzEntryPoint` can be used interchangeably and are both accepted and widely used for a function that is to be fuzzed. Hence, either way, I find them unfitting for this field as it refers to a file, that in theory can export multiple `fuzzTargets`/`fuzzEntrypoints` to begin with. 
Additionally, we have the `fuzzFunction` field, which is essentially, what the other two terms describe…

I'd propose these changes, which IMO makes the naming somewhat clearer:

* revert: `fuzzEntryPoint` ⇾ `fuzzTarget`
* rename `fuzzFunction` ⇾ `fuzzEntryPoint`

This would solve the issue of potentially confusing users by introducing "*new*" terminology 

However, that doesn't touch on the interface naming conflict issue. Then again, the newly introduced `FuzzTarget` is not really a "target". Wouldn't `FuzzMode` or similar be a better fit?

Reply:
Made changes as discussed.